<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client2_window</class>
 <widget class="QMainWindow" name="Client2_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrueCrypt - open file</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>551</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Write filename</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>154</x>
       <y>150</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Read</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>421</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>551</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>270</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>270</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <zorder>widget_2</zorder>
   <zorder>widget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
